index.html

```diff
@@ -479,6 +514,31 @@ <h3>Outline Button</h3>
             >
             </iframe>
           </div>
+
+          <div class="component-example">
+            <h3>Floating Button</h3>
+            <div class="component-example__container">
+              <button class="btn btn-primary btn-float">
+                <i class="fa fa-arrow-up" aria-hidden="true"></i>
+              </button>
+            </div>
+          </div>
+          <p class="code-snippet-heading">Code Snippets</p>
+          <div class="code-snippet">
+            <iframe
+              class="code-snippet-code"
+              src="https://carbon.now.sh/embed/JLgRdR0C7ihwMUcjZtHh"
+              style="
+                width: 100%;
+                height: 78px;
+                border: 0;
+                transform: scale(1);
```

Comment:
Try not to use inline styles

Reply:
Actually, every iframe has different heights. It would be difficult to give a unique class to every iframe. That's why it is done like this.